crates/sound/src/audio_backends/pw_backend.rs

```diff
@@ -4,25 +4,179 @@
 use super::AudioBackend;
 use crate::{Error, Result, SoundRequest};
 
-pub struct PwBackend {}
+use libspa_sys::spa_ringbuffer;
+use libspa_sys::spa_pod_builder;
+use libspa_sys::spa_pod_builder_state;
+use libspa_sys::spa_callbacks;
+use std::os::raw::c_void;
+use libspa_sys::spa_format_audio_raw_build;
+use libspa_sys::spa_audio_info_raw;
+use libspa_sys::SPA_AUDIO_FORMAT_S16;
+use libspa_sys::SPA_AUDIO_CHANNEL_FL;
+use libspa_sys::SPA_AUDIO_CHANNEL_FR;
+use libspa_sys::spa_pod;
+use libspa_sys::SPA_PARAM_EnumFormat;
+
+use pipewire as pw;
+use pw::prelude::*;
+use pw::{properties, spa};
+use pipewire::stream::Stream;
+use std::thread;
+use std::sync::{Mutex};
```

Comment:
```suggestion
use std::sync::Mutex;
```
Pretty sure rustfmt would fix that but just in case

Reply:
I'll correct that, thanks.